### DISCUSSION TITLE
Meaning of the name 'Hald'?

### DISCUSSION THREAD

**Discussion by Danack** [ImageMagick/ImageMagick#2953] [General] [answered]
Please can someone tell me the origin of the name 'Hald'?

I've been making an example of how to use them in Imagick, and wanted to explain where the name comes from, but all of my searches seem unable to find the origin of the name.

I'm not sure if it refers to a person, or is maybe some abbreviation/technical term.

**Comment by snibgo** [comment]
I have tried to find an answer to this, and failed.

**Comment by Danack** [comment, marked as answer]
Found it. The page at http://www.graphicsmagick.org/api/hclut.html has:

> The Hald CLUT algorithm has been developed by Eskil Steenberg as described at http://www.quelsolaar.com/technology/clut.html, 


I contacted Mr Steenberg, and he confirms that he named it after his grandmothers maiden name:

<img width="482" alt="Screen Shot 2020-12-11 at 12 04 56" src="https://user-images.githubusercontent.com/1505719/101901582-2919ca80-3ba9-11eb-8ced-e77c1bf6107a.png">

As it's a name, I guess it is 'Hald', not 'HALD'.

**Comment by snibgo** [comment]
Brilliant. Thanks for your research, and letting us know.
